src/razor/src/semantic/semanticTokensRangeHandler.ts

```diff
@@ -40,10 +49,33 @@ export class SemanticTokensRangeHandler {
         _semanticTokensParams: SerializableSemanticTokensParams,
         _cancellationToken: vscode.CancellationToken
     ): Promise<ProvideSemanticTokensResponse> {
+        let version = -1;
+        try {
+            const razorDocumentUri = vscode.Uri.parse(_semanticTokensParams.textDocument.uri);
+            const razorDocument = await this.documentManager.getDocument(razorDocumentUri);
+            if (razorDocument === undefined) {
+                return this.emptySemanticTokensResponse;
+            }
+
+            const textDocument = await vscode.workspace.openTextDocument(razorDocumentUri);
+            version = textDocument.version;
+            const synchronized = await this.documentSynchronizer.trySynchronizeProjectedDocument(
+                textDocument,
+                razorDocument.csharpDocument,
+                version,
```

Comment:
```suggestion
                version, // would have been more correct to have _semanticTokensParams.hostDocumentVersion,
```

For example `formattingParams` has `hostDocumentVersion` property but semantic tokens params does not, so I couldn't find a better version that's accessible here to send down to the synchronizer.

/cc @allisonchou @davidwengier @phil-allen-msft

Reply:
The version is sent by the language server, just looks like it's not included in the params object in typescript. Compare it to the params object defined in C#, the hostDocumentVersion property is missing.

Reply:
Applied a simpler fix by just including the missing `hostDocumentVersion` property to typescript params